wagtail/wagtailadmin/views/home.py

```diff
@@ -47,7 +46,13 @@ class RecentEditsPanel(object):
     def __init__(self, request):
         self.request = request
         # Last n edited pages
-        self.last_edits = PageRevision.objects.filter(user=self.request.user, created_at=F('page__latest_revision_created_at')).order_by('-created_at')[:5]
+        self.last_edits = PageRevision.objects.raw(
+            """
+            select wp.* FROM
+                wagtailcore_pagerevision wp JOIN (
+                    SELECT max(created_at) as max_created_at, page_id FROM wagtailcore_pagerevision where user_id = %s group by page_id
```

Comment:
Could you capitalise the SQL keywords: ``AS``, ``WHERE``, ``ON``, ``ORDER BY``, ``DESC``